ow_lander/scripts/arm_move_joints_guarded.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+from ow_lander import actions
+from ow_lander import node_helper
+
+import argparse
+
+parser = argparse.ArgumentParser(
+  formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+  description="Move all arm joints to either absolute or relative positions "
+              "and stop movement if resistance is encountered.")
+parser.add_argument('shoulder_yaw', type=float,
+  help="Shoulder yaw joint position/translation in radians")
```

Comment:
Should say "rotation" or "position/rotation." These definitely aren't translation joints. ROS refers to all joint displacements as "position" whether you're using a rotation or translation joint, but most robot joints tend to be rotational.

Reply:
Done.